### DISCUSSION TITLE
OpenSSL 3.3.x Support

### DISCUSSION THREAD

**Discussion by rubithedev** [prisma/prisma#24631] [Q&A] [answered]
Hello

Are we going to have OpenSSL 3.3.x support any time soon?

**Comment by ludralph** [comment, marked as answer]
Hi @Gabriel-Grechuk 👋 

Prisma has support for OpenSSL 3.x and any OpenSSL 3 minor versions. You can take a look at this related [blog](https://www.prisma.io/blog/wnip-q4-2022-f66prwkjx72s#improved-openssl-3x-support) post.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
